### DISCUSSION TITLE
Top langs don't work with my account

### DISCUSSION THREAD

**Discussion by Arthapz** [anuraghazra/github-readme-stats#1860] [Q&A] [answered]
Hi,
Top lang card don't seem to work with my account name

![image](https://user-images.githubusercontent.com/27437334/178276126-45e58de9-53fe-41e5-b5a2-e073b9c7862d.png)

```
[![Arthapz's Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Arthapz&layout=compact&show_icons=true&theme=github_dark&custom_title=Arthapz%27s%20Top%20Langs)](https://github.com/anuraghazra/github-readme-stats)
```

**Comment by rickstaa** [comment, marked as answer]
Hey, @Arthapz, thanks for your question. I checked the language card for your account, and it seems that it does not retrieve any data from the GraphQL API. This is because we currently only use data from non-forked repositories you own. For more information, see the [FAQ](#1770). Further, if you are using the public Vercel instance, it does not use data from your private repositories. Please deploy your own instance if you want to include data from private repositories (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).

**Reply by rickstaa** [reply]
@Arthapz Did that answer your question?

**Comment by DURGESH716** [comment]
hello,  @Arthapz, please try the below customized syntax for your profile:

 <p><img align="center" src="https://github-readme-stats.vercel.app/api/top-langs/?username=Arthapz&langs_count=8&theme=react")](https://github.com/Arthapz/github-readme-stats"/></p>

**Reply by DURGESH716** [reply]
try changing your username from the settings and then try, may be the problem is that your name and username are same

**Reply by DURGESH716** [reply]
try putting a username different from your name

**Reply by Arthapz** [reply]
i did, doesn't changed anything

**Reply by rickstaa** [reply]
@Arthapz Please see my comment above https://github.com/anuraghazra/github-readme-stats/discussions/1860#discussioncomment-3121536. Your user account currently contains no language data since we don't count forks (see https://github.com/anuraghazra/github-readme-stats/issues/1437). It only includes one empty repository and your user README, which is in markdown and is currently not detected by default (see https://github.com/github/linguist/issues/3964).

**Comment by mkredzel** [comment]
Hi @Arthapz, it seems to be working fine now that you have some repos in your account.

[![Arthapz's Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Arthapz&layout=compact&show_icons=true&theme=github_dark&custom_title=Arthapz%27s%20Top%20Langs)](https://github.com/anuraghazra/github-readme-stats)
